meta-refkit/conf/bblayers.conf.sample

```diff
@@ -20,6 +20,7 @@ REFKIT_LAYERS = " \
   ##OEROOT##/../meta-security-isafw \
   ##OEROOT##/../meta-intel-realsense \
   ##OEROOT##/../meta-clang \
+  ##OEROOT##/../meta-flatpak \
```

Comment:
Hmm... even 26ee25f is not enough. For some reason the bblayer.conf incompatiblity goes unnoticed and bitbake fails with missing flatpak.inc (since I don't have meta-flatpak enabled)